### PR TITLE
Add build_operators step to deploy edpm

### PR DESCRIPTION
**Pull request by viroel**
This PR has:
- [x] Appropriate testing (molecule, python unit tests)
- [x] Appropriate documentation (README in the role, main README is up-to-date)

### CODE REVIEW

File: ci_framework/playbooks/05-build-operators.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+- name: Run pre_operator_build hooks
+  vars:
+    hooks: "{{ pre_operator_build | default([]) }}"
+  ansible.builtin.import_playbook: ./hooks.yml
+
+- hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: CRC registry login
```
Comment (viroel) [inline ci_framework/playbooks/05-build-operators.yml:9]: Maybe we should set this in rhol_crc role

Comment (cjeanner) [reply ci_framework/playbooks/05-build-operators.yml:9]: Hmmm actually, THAT may be a hook - for instance, a pre_operator_build, managing the logging. Might be a nice thing as a follow-up - I'll check with the team if anyone has some interest in implementing that part :).

**Comment by cjeanner** [comment]
/lgtm

that's already a nice addition! Thanks @viroel !

**Pull request merged**